### DISCUSSION TITLE
2006/07/27/caption-%E5%92%8C-label-%E7%9A%84%E4%B9%A6%E5%86%99%E8%A7%84%E8%8C%83

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#99] [Announcements]
# 2006/07/27/caption-%E5%92%8C-label-%E7%9A%84%E4%B9%A6%E5%86%99%E8%A7%84%E8%8C%83

LabVIEW控件的 Caption 和 Label 的特性和用途很相似，都是给了控件一个有意义的名字。因此，在很多场合没有必要刻意区分他们。

https://blog.qizhen.xyz/2006/07/27/caption-%E5%92%8C-label-%E7%9A%84%E4%B9%A6%E5%86%99%E8%A7%84%E8%8C%83

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2006年07月28日 18:11 编辑
VB 已经记不清了，应该是差不多的。

回复
永青
2006年07月28日 09:34 编辑
看了半天，好像还是不太明白。。。
应该就跟vb里面的caption/name差不多吧？
